tests/ServiceTest.php

```diff
@@ -81,9 +85,16 @@ public function testBalance(): void
         $this->assertEquals($expectAmount, $actualBalance->getAmount());
         $this->assertEquals($expectCurrency, $actualBalance->getCurrency());
         $this->assertEquals(
-            "$expectAmount $expectCurrency",
+            "1,200.15 $expectCurrency",
             (string)$actualBalance
         );
+        $this->assertEquals(
```

Comment:
use asserArraySubset instead of assertEqueals for asserting arrays

Reply:
We are not checking subset. Checking subset will not fail test if we add some additional fields.

Reply:
yeah, you right